app/ui/window.js

```diff
@@ -215,8 +215,9 @@ module.exports = class Window {
     for (const ev of ['maximize', 'unmaximize', 'minimize', 'restore']) {
       window.on(ev, () => rpc.emit('windowGeometry change'));
     }
-    rpc.win.on('move', () => {
-      rpc.emit('move');
+    rpc.window.on('move', () => {
```

Comment:
```suggestion
    window.on('move', () => {
```

I think that @juancampa suggested to use `window` variable that is directly accessible instead of `rpc.win.on()`

Reply:
Yeah. Just updated for window.